src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -762,7 +762,24 @@ export default function ({
         break;
       default:
         if (nonEmptyToken && nonEmptyToken.type.indexOf('url') < 0) {
-          context.prefixToAdd = ', ';
+          if (
+            // if not on the first line
+            context.rangeToReplace &&
+            context.rangeToReplace.start &&
+            context.rangeToReplace.start.lineNumber > 1
+          ) {
+            // save original start position
+            const startColumn = context.rangeToReplace.start.column;
+            const prevLineNumber = context.rangeToReplace.start.lineNumber - 1;
+            const prevLineLength = context.editor?.getLineValue(prevLineNumber).length ?? 0;
+            // go back to the end of the previous line
+            context.rangeToReplace = {
+              start: { lineNumber: prevLineNumber, column: prevLineLength + 1 },
+              end: { ...context.rangeToReplace.end },
+            };
+            // add a comma at the end of the previous line, a new line and indentation
+            context.prefixToAdd = (prevLineLength < 6 ? '\n' : ',\n') + ' '.repeat(startColumn - 1);
```

Comment:
6 seems like a magic number here. Could you extract it into a named constant, or add a comment to explain why 6, and not another number? This will be useful information for someone trying to debug any problems related to this code in the future. Thanks!